src/main/java/HandleTruth.java

```diff
@@ -1,37 +1,58 @@
+import java.io.Serializable;
 import java.util.*;
 
 public class HandleTruth {
-    public static void wordCount() {
-        String msg = "son we live in a world that has walls and those walls have to be guarded by men with guns whos gonna do it you you lieutenant weinberg i have a greater responsibility than you can possibly fathom you weep for santiago and you curse the marines you have that luxury you have the luxury of not knowing what i know that santiagos death while tragic probably saved lives and my existence while grotesque and incomprehensible to you saves lives you dont want the truth because deep down in places you dont talk about at parties you want me on that wall you need me on that wall we use words like honor code loyalty we use these words as the backbone of a life spent defending something you use them as a punchline i have neither the time nor the inclination to explain myself to a man who rises and sleeps under the blanket of the very freedom that i provide and then questions the manner in which i provide it i would rather you just said thank you and went on your way otherwise i suggest you pick up a weapon and stand a post either way i dont give a damn what you think you are entitled to";
-
-        // split the message into word set
-        String[] words = msg.split(" ");
-
-        // use map to count the occurrences of each word
-        HashMap<String, Integer> wordMap = new HashMap<>();
-        for (String word : words) {
-            if (wordMap.containsKey(word)) {
-                wordMap.put(word, wordMap.get(word) + 1);
-                continue;
+
+    public static Map<? extends Serializable,? extends Object> phrase(String phrase) {
+
+        {
+            String msg = "son we live in a world that has walls and those walls have to be guarded by men with guns whos gonna do it you you lieutenant weinberg i have a greater responsibility than you can possibly fathom you weep for santiago and you curse the marines you have that luxury you have the luxury of not knowing what i know that santiagos death while tragic probably saved lives and my existence while grotesque and incomprehensible to you saves lives you dont want the truth because deep down in places you dont talk about at parties you want me on that wall you need me on that wall we use words like honor code loyalty we use these words as the backbone of a life spent defending something you use them as a punchline i have neither the time nor the inclination to explain myself to a man who rises and sleeps under the blanket of the very freedom that i provide and then questions the manner in which i provide it i would rather you just said thank you and went on your way otherwise i suggest you pick up a weapon and stand a post either way i dont give a damn what you think you are entitled to";
+
+            // split the message into word set
+           // String[] words = msg.split(" ");
```

Comment:
remove command if not used. do not leave it as a comment. 

also, if this is removed, why is the msg variable still left?